lib/Stan_math_code_gen.ml

```diff
@@ -157,24 +164,20 @@ let pp_statement pp_statement_with_meta ppf s =
   | MarkLocation _ -> () (* XXX *)
   | Check _ -> () (* XXX *)
   | IfElse (cond, ifbranch, elsebranch) ->
-      let pp_else ppf x =
-        pf ppf " else {\n %a;\n}" pp_statement_with_meta x
-      in
-      pf ppf "if (%a) %a %a\n" pp_expr cond pp_block
-        (pp_statement_with_meta, ifbranch)
+      let pp_else ppf x = pf ppf " else {\n %a;\n}" pp_statement x in
```

Comment:
[question]
Aren't we supposed to get rid of all the hard `\n` instances in favor of their breaks?

Reply:
Yeah, I think so. This was just a formatting change - the code here is left over from before I knew that about the Format library. (I was going to say "before I knew the library" but I still am learning a lot). I'll go ahead and change it.